<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reg_window</class>
 <widget class="QDialog" name="reg_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>notes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recource.qrc">
    <normaloff>:/resource/images/заголовок-10.png</normaloff>:/resource/images/заголовок-10.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(120, 65, 81);
</string>
  </property>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>300</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 8pt &quot;Montserrat&quot;;
background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
   <property name="text">
    <string>Придумайте логин</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Pass">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 8pt &quot;Montserrat&quot;;
background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
   <property name="text">
    <string>Придумайте пароль</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Confirm">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>460</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 8pt &quot;Montserrat&quot;;
background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
   <property name="text">
    <string>Повторите пароль</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>245</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recource.qrc">:/resource/images/иконки_Log_pas2-03.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Pass_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>325</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="pixmap">
    <pixmap resource="recource.qrc">:/resource/images/иконки_Log_pas2-02.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_Confirm_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: italic 12pt &quot;Montserrat&quot;;
background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recource.qrc">:/resource/images/повтор-09.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>-140</y>
     <width>1041</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recource.qrc">:/resource/images/писанина2-04.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-width: 0px; 
border-style: solid; 
border-color: white;
border-bottom-width: 1px;
background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-width: 0px; 
border-style: solid; 
border-color: white;
border-bottom-width: 1px;
background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmLineEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>220</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-width: 0px; 
border-style: solid; 
border-color: white;
border-bottom-width: 1px;
background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="registerPushButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>530</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 1px;
font: italic 12pt &quot;Montserrat&quot;;
background-color: rgb(255, 255, 255);
border-color: white;
border-radius: 45px;</string>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Error">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>1061</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>510</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.5, y1:0.869318, x2:0.499, y2:1, stop:0.741573 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="recource.qrc">:/resource/images/тень-05.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>confirmLineEdit</zorder>
  <zorder>label_Name</zorder>
  <zorder>label_Pass</zorder>
  <zorder>label_Confirm</zorder>
  <zorder>label_Confirm_2</zorder>
  <zorder>label_Name_2</zorder>
  <zorder>label_Pass_2</zorder>
  <zorder>passwordLineEdit</zorder>
  <zorder>nameLineEdit</zorder>
  <zorder>label_Error</zorder>
  <zorder>label_2</zorder>
  <zorder>registerPushButton</zorder>
 </widget>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>confirmLineEdit</tabstop>
  <tabstop>registerPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="recource.qrc"/>
 </resources>
 <connections/>
</ui>
